package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "dog_lover",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@mantine/core": "^4.2.4",
+    "@mantine/dates": "^4.2.4",
+    "@mantine/form": "^4.2.4",
+    "@mantine/hooks": "^4.2.4",
+    "@mantine/prism": "^4.2.4",
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^13.0.0",
+    "@testing-library/user-event": "^13.2.1",
+    "@types/jest": "^27.0.1",
+    "@types/node": "^16.7.13",
+    "@types/react": "^18.0.0",
+    "@types/react-dom": "^18.0.0",
+    "@types/react-router-dom": "^5.3.3",
+    "dayjs": "^1.11.2",
```

Comment:
Is this used somewhere?

Reply:
Are you referring to the ```daysjs```? If yes, I do not use it anywhere but it was downloaded when I installed 
Mantine and I left it there!